lib/Backend/Lower.cpp

```diff
@@ -100,6 +100,40 @@ Lowerer::Lower()
     }
 
     this->LowerRange(m_func->m_headInstr, m_func->m_tailInstr, defaultDoFastPath, loopFastPath);
+
+#if DBG
+    // TODO: (leish)(swb) implement for arm
+#if defined(_M_IX86) || defined(_M_AMD64)
+    if (CONFIG_FLAG(ForceSoftwareWriteBarrier))
+    {
+        // find out all write barrier setting instr, call Recycler::WBSetBit for verification purpose
+        // should do this in LowererMD::GenerateWriteBarrier, however, can't insert call instruction there
+        IR::Instr* instr = m_func->m_headInstr;
+        while (instr != m_func->m_tailInstr)
+        {
+            if (instr->m_src1 && instr->m_src1->IsAddrOpnd())
+            {
+                IR::AddrOpnd* addrOpnd = instr->m_src1->AsAddrOpnd();
+                if (addrOpnd->GetAddrOpndKind() == IR::AddrOpndKindWriteBarrierCardTable)
+                {
+                    auto& lea = instr->m_prev->m_prev;
```

Comment:
sure, I can do it in next commit

---
In reply to: [92857326](https://github.com/Microsoft/ChakraCore/pull/2222#discussion_r92857326) [](ancestors = 92857326)